stargate-core/app/com/salesforce/mce/stargate/models/McSsoDecodedJwt.scala

```diff
@@ -51,14 +51,18 @@ object McSsoJwtRequest {
 case class McSsoJwtRequestRest(
   authEndpoint: String,
   apiEndpointBase: String,
-  refreshToken: String
+  refreshToken: String,
+  mcAccessToken: Option[String] = None,
```

Comment:
since this is no longer part of the description in the doc linked above. Can you add some comments here describing those two additions?

Reply:
@realstraw - added a scaladoc section for the case class McSsoJwtRequestRest which notes what they are and that they are not part of the official documentation doc.

Reply:
Don't think the default initialization of `None` would do anything here. play-json is going to put them as non if absent from the json string anyway.

Reply:
Someone had asked for that to be set to None but I agree - doesn't make sense. I proved it was set to None when not provided by adding a couple tests in a new spec in stargate-core:  McSsoDecodedJwtSpec

Reply:
Thanks @jeffzickgraf